tests/unit/python/test_python_submissions.py

```diff
@@ -247,97 +247,3 @@ def test_build_job_spec_with_post_hooks(self, mock_api_client):
         assert len(result["tasks"]) == 2
         assert result["tasks"][1]["task_key"] == "task_b"
         assert result["tasks"][1]["new_cluster"]["spark_version"] == "14.3.x-scala2.12"
-
-    @patch("dbt.adapters.databricks.python_models.python_submissions.DatabricksApiClient")
-    def test_build_job_spec_with_post_hooks_serverless_job_cluster(self, mock_api_client):
```

Comment:
Removing these since the logic to muck around with the cluster settings in additional tasks was removed [here](https://github.com/databricks/dbt-databricks/pull/762/commits/38c3ce48fd0f76bb1cad5604180de7f174dea8db)